package.json

```diff
@@ -3,13 +3,12 @@
   "version": "0.1.0",
   "description": "Knative website",
   "main": "index.js",
-  "dependencies": {},
   "devDependencies": {
     "autoprefixer": "^9.8.6",
     "js-yaml": "^3.14.0",
     "lodash": ">=4.17.19",
     "mixin-deep": ">=1.3.2",
-    "postcss-cli": "^6.1.3",
+    "postcss-cli": "^8.3.1",
```

Comment:
@vaikas did you tested the website locally and there is not problems with new version?

I asked because the bump is a major semantic change which usually comes with braking APIs

Reply:
This particular tool is only used for processing CSS before shipping, and should be okay.